### ISSUE TITLE
Claims to update hosts but no sir

### ISSUE THREAD

**Issue by jonathan-dejong** [bradp/vvv-provision-flipper#1]
This is an awesome addition to VVV.. it speeds up my site creation from 5-10+min to ~1min.

However there seem to be an issue with writing to the hosts file.
It claims to do it in the log just like an ordinary vagrant up but it doesn't. It will also not ask for password even if it should  which I suspect is part of the reason why it fails. 

To reproduce:
- Setup a new site/project inside www with the vvv-init.sh, vvv-nginx.conf and vvv-hosts files.
- Run `flip set quick`
- Run `vagrant provision`

It will log things like
`
==> default:  * Added vvv.dev from /srv/www/vvv-hosts
==> default:  * Added local.wordpress.dev from /srv/www/vvv-hosts
==> default:  * Added local.wordpress-trunk.dev from /srv/www/vvv-hosts
==> default:  * Added src.wordpress-develop.dev from /srv/www/vvv-hosts
==> default:  * Added build.wordpress-develop.dev from /srv/www/vvv-hosts
`
and the new site but the new site will not be added to the hosts file if checked manually afterwards (it will also not work in the browser). 
`sudo nano /private/etc/hosts`

**Comment by bradp** [comment]
If you do `flip reset` do you also get this issue?

**Comment by jonathan-dejong** [comment]
No, running it with the regular script it works.. It's just when setting `flip set quick`
